Readme.md

```diff
@@ -3,19 +3,30 @@
 [![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/T190812)
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
-<!-- default file list -->
-*Files to look at*:
+# Spreadsheet for ASP.NET Web Forms - How to save/load documents from/to a database
 
-* [Default.aspx](./CS/ASPxSpreadsheetBinding/Default.aspx) (VB: [Default.aspx](./VB/ASPxSpreadsheetBinding/Default.aspx))
-* **[Default.aspx.cs](./CS/ASPxSpreadsheetBinding/Default.aspx.cs) (VB: [Default.aspx.vb](./VB/ASPxSpreadsheetBinding/Default.aspx.vb))**
-<!-- default file list end -->
-# ASPxSpreadsheet - How to save and load documents from a database
+This example demonstrates how to configure the [Spreadsheet control](https://docs.devexpress.com/AspNet/16157/components/spreadsheet) to work with a database. 
+
+![Connect Spreadsheet to Database](save-to-database.gif)
+
+## Overview
+
+During page initialization, load a document from a database as a byte array and pass it to the control's [Open](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxSpreadsheet.ASPxSpreadsheet.Open(System.String-DevExpress.Spreadsheet.DocumentFormat-System.Func-System.Byte---)) method to open the document in the Spreadsheet.
 
+The [Saving](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxSpreadsheet.ASPxSpreadsheet.Saving) event occurs when a user clicks the built-in **Save** or **SaveAs** command on the Spreadsheet's ribbon. To save changes, handle the `Saving` event and update the data source. To save the document after a user clicks an external button, update the database in the button's [Click](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxButton.Click?p=netframework) event handler.
```

Comment:
First..

> when a user clicks the built-in **Save** or **SaveAs** command on the Spreadsheet's **ribbon**.

It's more precise to say as follows:

> when a user clicks the built-in **Save** or **SaveAs** command in the Spreadsheet's Ribbon UI.

Second...

> To save the document after a user clicks an external button

Of what "external" button do we speak here? Do you mean a custom button that is not a part of the (automatically generated) Ribbon UI?

Therefore, my take on this paragraph is the following:

```suggestion
The [Saving](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxSpreadsheet.ASPxSpreadsheet.Saving) event occurs when a user clicks the built-in **Save** or **SaveAs** command in the Spreadsheet's Ribbon UI. To save changes, handle the `Saving` event and update the data source. To save the document after a user clicks a custom button, update the database in the button's [Click](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxButton.Click?p=netframework) event handler.
```